tests/suites/charmhub/download.sh

```diff
@@ -15,6 +15,23 @@ run_charmhub_download() {
 	destroy_model "${name}"
 }
 
+run_charmhub_download_with_resources() {
+	echo
+	name="charmhub-download-with-resources"
+
+	file="${TEST_DIR}/${name}.log"
+
+	ensure "${name}" "${file}"
+
+	output=$(juju download juju-qa-test-resources --resources --filepath="${TEST_DIR}/juju-qa-test-resources.charm" 2>&1 || true)
+	check_contains "${output}" 'Fetching charm "juju-qa-test-resources"'
```

Comment:
no check for the resources in the output

Reply:
Good point.

It is checking the output (in a round about way), but that is only ensured because the test charm doesn't print the `amd64-linux` or the `arm64-linux`, rather it is done by the resource downloaded from charmhub.

I'll add another check and add some comments.